### ISSUE TITLE
Provide a documented way to turn off EPS handling via ghostscript

### ISSUE THREAD

**Issue by raphaelm** [python-pillow/Pillow#7391]
Pillow parses EPS files through ghostscript.

Ghostscript is troubled with regular security issues of critical impact, such as remote code execution. There has been [this one](https://github.com/farisv/PIL-RCE-Ghostscript-CVE-2018-16509), two CVEs this year already, and there is another one [just coming up](https://git.ghostscript.com/?p=ghostpdl.git;a=commit;h=e59216049cac290fb437a04c4f41ea46826cfba5) with the patch not yet included in any release.

We do not need EPS handling and we can turn it off in our code using `Image.open(…, formats=["JPG", "PNG", …])`. However, we use third-party components that also use Pillow (such as `reportlab`) and we'd like to make sure they also don't accidentally open EPS files and we can't modify the code.

From what I can tell, we can accomplish this with:

```
from PIL import Image

if "EPS" in Image.ID:
    Image.ID.remove("EPS")
```

However, that seems to be undocumented and feels like unclean monkeypatching. I think it would be worth it providing a documented API to turn off third-party backends like ghostscript globally.

**Comment by radarhere** [comment]
Hi. See what you think of #7392. It adds `EpsImagePlugin.gs_binary`, so that setting `EpsImagePlugin.gs_binary = False` would result in `OSError: Unable to locate Ghostscript on paths` when trying to load an EPS image.

**Comment by raphaelm** [comment]
Wow, that was fast! Not sure if I read the code exactly, but wouldn't this trigger OSError from line 72 whenever I set it to `False` and import the module?

**Comment by radarhere** [comment]
No, the `Ghostscript` class is only instantiated when an image is loaded.
https://github.com/python-pillow/Pillow/blob/a84a43528ebf8756369a3b7248312c4122ef3bc9/src/PIL/EpsImagePlugin.py#L383-L386

With the workaround that you posted above, an EPS file won't be identified. In my suggestion, it will be identified. `Image.open` will complete successfully. It will just raise an OSError once `load()` is called on it (which many Pillow operations call internally).

**Comment by raphaelm** [comment]
Ah sorry, I've mistaken `Ghostscript` to be a class. My bad! Yes, sounds good then. (A note in the docs about it would be great to make it "official" so I can somehow expect it to continue to work in future versions.)

**Comment by radarhere** [comment]
Sure. I've added a commit to the PR. You can see the preview at https://pillow--7392.org.readthedocs.build/en/7392/handbook/image-file-formats.html#eps
> To use Ghostscript, Pillow searches for the “gs” executable. On Windows, it also searches for “gswin32c” and “gswin64c”. If you would like to customise this behaviour, `EpsImagePlugin.gs_binary = "gswin64"` will set the name of the executable to use. `EpsImagePlugin.gs_binary = False` will prevent Ghostscript from being used altogether.

**Comment by raphaelm** [comment]
Thank you <3
